final/pkg/handle/handler.go

```diff
@@ -0,0 +1,74 @@
+package handle
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/NikhilSharmaWe/gomodule/final/pkg/database"
+)
+
+func CreateBook(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "text/html; charset=utf-8")
+	fmt.Fprint(w, `
+	<form action="/" method="GET">
```

Comment:
html templates are not required. There will a UI in most of the cases, so you don't need to worry about html.